tests/utils.py

```diff
@@ -61,7 +62,14 @@ def skip_fail_test(self, message):
 # Define the testing paths
 testing_path = os.path.abspath(os.path.dirname(__file__))
 testing_input = os.path.join(testing_path, "reference-files")
-testing_temp = os.path.join(testing_path, "testing-tmp")
+# if testing directory is specified use it
+testing_temp = get_env_variable("MESHPY_TESTING_DIR")
+if testing_temp == "default_not_set":
+    # create default one if not specified
+    testing_temp = os.path.join(testing_path, "testing-tmp")
+else:
+    # convert string to path
+    testing_temp = Path(testing_temp)
```

Comment:
@isteinbrecher 
I think it's on the long run cleaner to specify the directory from a variable and provide a default location.